src/Browscap/Coverage/Processor.php

```diff
@@ -80,7 +80,7 @@
      * A storage variable of the pattern ids covered by tests for a specific file (set when processing of that
      * file begins)
      *
-     * @var string[]
+     * @var array
```

Comment:
I'd rather be more specific about the contents of this: `string[]` is better (if that's what it is)